### PR TITLE
Add PGOPTIONS to Dockerfiles

### LINKED ISSUE #543: Add PGOPTIONS env var in Dockerfile
For `Dockerifle` and `Dockerfile.dev`: Add: 

```
 ENV PGOPTIONS "-c search_path=auth"
 ```

### PR DESCRIPTION
**Pull request by komninoschat**
Fixes #543.

### CODE REVIEW

**Review by nunopato — COMMENTED** [review]
Do you need those here if they are present on the `.env.*` files?

**Comment by elitan** [comment]
> Do you need those here if they are present on the `.env.*` files?

The point is to not have this something like ``PGOPTIONS "-c search_path=auth"` in any `.env*` file. Instead, we have it in the Dockerfile directly.

**Review by elitan — APPROVED** [review]
Wait for @nbp to comment or approve before merging.

**Pull request merged**
